gensim/models/base_any2vec.py

```diff
@@ -1181,14 +1280,14 @@ def _log_progress(self, job_queue, progress_queue, cur_epoch, example_count, tot
             logger.info(
                 "EPOCH %i - PROGRESS: at %.2f%% examples, %.0f words/s, in_qsize %i, out_qsize %i",
                 cur_epoch + 1, 100.0 * example_count / total_examples, trained_word_count / elapsed,
-                utils.qsize(job_queue), utils.qsize(progress_queue)
+                -1 if job_queue is None else utils.qsize(job_queue), utils.qsize(progress_queue)
```

Comment:
Can be confusing, should be clarified in the documentation

Reply:
done